### ISSUE TITLE
Ajout de test unitaire pour l'intégralité des Variants du projet

### ISSUE THREAD

**Issue by fraxken** [fraxken/Node-Estree#6]
L'objectif est d'ajouter des tests unitaires pour l'intégralité des variants présents dans le projet. Un Variant est une entité (un node) défini par la Spécification [ESTree](https://github.com/estree/estree). Ils servent à décrire un élément précis dans un code JavaScript!

Exemple avec Variant **Literal**
```ts
interface Literal extends Expression {
    type: "Literal";
    value: string | boolean | null | number | RegExp;
}
```

Le Literal(5) sera donc par exemple égal à `5` (tout simplement). Evidemment il y a des Variants plus complexe que les autres. Par exemple BlockStatement() produira:

```js
{ // <-- un block ^^

}
```

Un ensemble de tests a déjà été réalisé pour une dizaine de variants:
- https://github.com/fraxken/Node-Estree/blob/master/test/Identifier.spec.ts
- https://github.com/fraxken/Node-Estree/blob/master/test/ForOfStatement.spec.ts
- https://github.com/fraxken/Node-Estree/blob/master/test/AwaitExpression.spec.ts

La liste complète des Variants est ici: https://github.com/fraxken/Node-Estree/blob/master/src/estree/es5.ts#L9

Je vous recommande fortement l'utilisation de l'outil https://astexplorer.net/ pour faire des tests si vous n'êtes pas à l'aise.

Vidéo pouvant vous aider à mieux comprendre:
- https://www.youtube.com/watch?v=zSYrEbggqWA&feature=youtu.be
- https://www.youtube.com/watch?v=p-weX1QZNo4 (live coding)
- https://www.youtube.com/watch?v=Bi_lFJa8msE (live coding)

**Comment by fraxken** [comment]
Note supplémentaire: Vous n'êtes pas l'obligation de PR l'intégralité des variants (vous pouvez en faire déjà un si c'est votre première PR).

**Comment by Spiquet** [comment]
Hello @fraxken,  I would like to make my first contribution to the project. Can I work on this issue?

**Comment by fraxken** [comment]
@Spiquet Yes no problem, you'r welcome. If you need to discuss or help do not hesitate to join this discord: https://discord.gg/4Wn8rjAtB4

**Comment by Spiquet** [comment]
Thanks @fraxken.
